sdk/core/core-client/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.1.3 (Unreleased)
 
+- Fix an issue of lost properties when flattening array in deserialization [issue 15653](https://github.com/azure/azure-sdk-for-js/issues/15653)
```

Comment:
NIT: perhaps consider having a "Key Bug Fixes" subtitle.

Reply:
yes! the new convention.